### PR TITLE
feat: add student, professor, and aggregation dashboards

### PR DESCRIPTION
**Pull request by Ian2012**
## Description

This PR creates the three minimum dashboards for Unidigital:

- Aggregation Dashboard:
image
- Professor Dashboard:
image
image
image
- Student Dashboard:
image

### CODE REVIEW

File: tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      parents:
+      - ROOT_ID
+      - GRID_ID
+      - TABS-UMFjQI6rV3
+      type: TAB
+    TAB-E0yEilj5zM:
+      children: []
+      id: TAB-E0yEilj5zM
+      meta:
+        defaultText: Tab title
```
Comment (felipemontoya) [inline tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml]: Do this default strings need to be there?

File: tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+      - GRID_ID
+      id: ROOT_ID
+      type: ROOT
+    TAB-2Gg6rnNmk:
+      children: []
+      id: TAB-2Gg6rnNmk
+      meta:
+        defaultText: Tab title
+        placeholder: Tab title
+        text: "Comparaci\xF3n de cursos"
```
Comment (felipemontoya) [inline tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml]: Can you write the accents normally? or do they always need to be escaped?

File: tutoraspects_unidigital/patches/superset-extra-assets [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{% include 'openedx-assets/Aggregation_Dashboard_43.yaml' %}
```
Comment (felipemontoya) [inline tutoraspects_unidigital/patches/superset-extra-assets]: Does the _43, _44, _45 suffix mean something? if I don't have access to the previous 42 does it make a difference if we remove them?

I think we could benefit from a prefix like campus_ or unidigital_ better than the number.

File: tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml (hunk 1 above)
Comment (Ian2012) [reply tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml]: Not entirely sure, those are in the aspects assets by default too

File: tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml (hunk 2 above)
Comment (Ian2012) [reply tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml]: Those were exported like that, and the translation process also scape those: 

https://github.com/openedx/tutor-contrib-aspects/pull/242/files#diff-4a3d72c953a859f523f162da917e1141a825d2ef96d081a3e0732fb4d7d7d19cR34

File: tutoraspects_unidigital/patches/superset-extra-assets (hunk 3 above)
Comment (Ian2012) [reply tutoraspects_unidigital/patches/superset-extra-assets]: No, we can remove the prefix, it was just to keep the standard.

Is there really a benefit for that? I can rename the folder from `openedx-assets` to `openedx-assets-unidigital`

File: tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml (hunk 2 above)
Comment (Ian2012) [reply tutoraspects_unidigital/templates/openedx-assets/Aggregation_Dashboard_43.yaml]: To align with the Aspects implementation, the text will be in English and proper translations will be added.

**Pull request merged**